### PR TITLE
docs: add systemd timer docs

### PR DESCRIPTION
**Pull request by alxndr13**
related: https://github.com/inovex/CalendarSync/issues/26

### CODE REVIEW

File: docs/systemd-timers.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Running CalendarSync periodically using systemd
+
+To run CalendarSync periodically / automatically on specific times using systemd, two files are necessary.
+
+- A [service unit](https://www.freedesktop.org/software/systemd/man/systemd.service.html) file, which we will call: `CalendarSync.service`
```
Comment (MichaelEischer) [inline docs/systemd-timers.md:5]: Nitpick, `, which we will call` is somewhat redundant. Replacing it with `called` or removing it entirely would also be fine.

File: docs/systemd-timers.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+```systemd
+[Unit]
+Description=Run CalendarSync
+
+[Service]
+ExecStart=/path/to/binary/calendarsync --config path/to/your/sync.yaml --storage-encryption-key $key
+```
+
+The following content should be placed under: `.config/systemd/user/CalendarSync.timer`:
```
Comment (MichaelEischer) [inline docs/systemd-timers.md:18]: the colon after `under:` is redundant

**Review by MichaelEischer — APPROVED** [review]
I've only got two minor comments, see below

File: docs/systemd-timers.md (hunk 2 above)
Comment (alxndr13) [reply docs/systemd-timers.md]: fixed :)

File: docs/systemd-timers.md (hunk 1 above)
Comment (alxndr13) [reply docs/systemd-timers.md]: removed 👍🏼

**Pull request merged**